src/remoted/shared_download.c

```diff
@@ -571,6 +571,11 @@ void w_create_group(char *group){
     }
 }
 
+/* Parse files.yml file
+ * Return 1 on parse success.
+ * Return 0 if no parse was performed (missing file).
+ * Return -1 on parse error.
+*/
 int w_prepare_parsing()
```

Comment:
Hi @chemamartinez,

That function itself and the callee procedures already make the necessary checks. For instance:
- Error if the _files.yml_ exists but couldn't be opened.
- Error if the file syntax is invalid.

That's why there is no error checking here. In the particular case of the first initialization, we need to wait for Modulesd to start if the file was successfully loaded. Otherwise, `w_prepare_parsing()` produces the necessary logs and cleanup.

Cheers.

Reply:
Makes sense, I'll accept the PR.

Thank you @vikman90 !